tests/unit/test_cli.py

```diff
@@ -18,7 +18,7 @@ def test_cli_happy_path(single_passing_xml, mocker):
     # Setup
     env_vars = {'QTEST_API_TOKEN': 'valid_token'}
     project_id = '12345'
-    test_cycle_name = 'RPC_PRODUCT_RELEASE'
+    test_cycle_name = 'pike'
```

Comment:
The commit comment does not indicate why this value has been changed. Can you indicate why?

Reply:
addressed in new commit message.